IAmALamp/assets/images/icons.ts

```diff
@@ -0,0 +1,21 @@
+import aboutIcon from './about-icon.png';
```

Comment:
There is already an icon component under components, take a look there, we could use either one, but I would like to avoid duplication.

Reply:
Oh, sorry, I hadn't seen the icon component. I've updated the Icon list with new icons and removed the List Icon component